addons/hr_employee_hourly_wage/__manifest__.py

```diff
@@ -0,0 +1,25 @@
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+
+{
+    'name': 'Employee Hourly Wage',
+    'version': '1.0',
+    'category': 'Services/Employee Hourly Wage',
+    'summary': 'Employee Hourly Wage',
+    'description': """
+This module assigns an hourly wage to employees to be used by other modules.
+============================================================================
+
+    """,
+    'depends': ['hr'],
+    'installable': True,
```

Comment:
Do we want to allow the user to install it from the Web UI?

Reply:
No, probably not :)

Reply:
So it doesn't install as a dependency either if I disable this option :)